ui/raidboss/raidboss_options.ts

```diff
@@ -93,6 +94,7 @@ const defaultRaidbossConfigOptions = {
   TimelineLanguage: undefined as (Lang | undefined),
   TimelineEnabled: true,
   AlertsEnabled: true,
+  DefaultPlayerLabel: undefined as (PartyMemberParamObjectKeys | undefined),
```

Comment:
Can this be `undefined`? 

```suggestion
  DefaultPlayerLabel: 'shortName' as PartyMemberParamObjectKeys,
```

Reply:
Since I'm not as familiar with how the config portion works, I elected to follow the pattern used above for `AlertsLanguage` and `TimelineLanguage`. If that's not required, then would it even need the `as` cast at all, e.g. following the `DefaultAlertOutput` example above instead?

Reply:
Oh, `AlertsLanguage` and `TimelineLanguage` can be `undefined`.  If they are set to default, the `setterFunc` in `raidboss_config.ts` sets them to be `undefined`.  `DefaultPlayerLabel` should never be `undefined`, I think?